src/main/java/com/mraof/minestuck/item/weapon/OnHitEffect.java

```diff
@@ -314,6 +322,25 @@ static OnHitEffect enemyPotionEffect(Supplier<MobEffectInstance> effect)
 		return (stack, target, attacker) -> target.addEffect(effect.get());
 	}
 	
+	static OnHitEffect bossBuster()
+	{
+		return (stack, target, attacker) -> {
+			if(attacker instanceof ServerPlayer player && target instanceof UnderlingEntity) {
```

Comment:
UnderlingEntity seems a little too broad for a boss injuring weapon

If its for dealing with powerful boss mobs, we may want to create an entity tag for entities who apply. Either that or feed in a supplier with a specific entity type (such as is done in targetSpecificAdditionalDamage()) to allow the entity to be tailored to each weapon that could end up using this.